kuzzleio/documentation
# [feature] Fix indent in imported code snippets

## What does this PR do?
Automatically fix the indentation of imported code snippets (useful when importing partial snippet files).

### How should this be manually tested?
import a code-block with significant (consistent) indentation from a snippet file, like

```javascript
function() {
  foo.forEach(f => {
    /* snippet:start */
    console.log(f)
    /* snippet:end */
  })
}
```
See how the imported snippet is 
```javascript
console.log(f) // Look ma! No spaces at the beginning of the line!
```
### Other changes
Switched to `vuepress-frontmatter-lint` package.
